rootfs/apply.go

```diff
@@ -96,11 +97,12 @@ func Prepare(snapshots Snaphotter, mounter Mounter, layers []ocispec.Descriptor,
 	)
 
 	for _, layer := range layers {
+		layerDigest := digest.Digest(layer.Digest)
```

Comment:
Please don't reassign: this fix needs to be upstreamed to OCI. I'll submit a PR today.